core/local/build.gradle.kts

```
@@ -0,0 +1,13 @@
+plugins {
+    alias(libs.plugins.team.noweekend.android.library)
+    alias(libs.plugins.team.noweekend.hilt)
+}
+
+android {
+    namespace = "team.noweekend.core.local"
+}
+
+dependencies {
+    implementation(project(":core:common-android"))
+    implementation(project(":core:common-kotlin"))
```

```suggestion
    implementation(project(":core:model"))
    implementation(project(":core:common-kotlin"))
```
p2 : local 모듈에서는 remote랑은 다르게 model 모듈에 대한 의존이 없어도 괜찮을까용?
마찬가지로 local 모듈에서 common-android를 바라보는 경우가 어떤 경우가 있을까요?!

---

remote 나 local이 둘다 같은 model을 의존하게 해야할꺼 같당

---

아 잘못적었네 ㅋㅋ model이 아니라 common:android 였음

---

로컬 모델에서 뭔가 도메인으로 들어갈때 도메인 모델에 대한 매퍼가 필요힐 수도 있다고 생각해서 넣어 놓긴했어 라고 답을 하려고 했는데 안들어가져 있군? 이거는 의존성 다시 추가하겠습니다 ㅋ_ㅋ

android 모듈 의존성은 필요없을거 같기도 하네..? 근데 있을 수도 있다고 생각하긴해 common인 만큼? 그래도 아예 배제 해놓는게 나을까?

---

위 리뷰에 달아놓은 것 처럼 android는 없어도 될 듯한 느낌입니당~